src/wallet/wallet.cpp

```diff
@@ -2737,17 +2737,19 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
         return false;
     }
 
+    uint256 tip_hash;
+    int tip_height;
     CMutableTransaction txNew;
-
-    txNew.nLockTime = GetLocktimeForNewTransaction(chain(), locked_chain);
-
     FeeCalculation feeCalc;
     CAmount nFeeNeeded;
     int nBytes;
     {
         std::set<CInputCoin> setCoins;
         auto locked_chain = chain().lock();
         LOCK(cs_wallet);
+        tip_hash = GetLastBlockHash();
+        tip_height = GetLastBlockHeight();
```

Comment:
`tip_hash` and `tip_height` could be `const`:
```suggestion
    CMutableTransaction txNew;
    FeeCalculation feeCalc;
    CAmount nFeeNeeded;
    int nBytes;
    {
        std::set<CInputCoin> setCoins;
        auto locked_chain = chain().lock();
        LOCK(cs_wallet);
        const uint256 tip_hash = GetLastBlockHash();
        const int tip_height = GetLastBlockHeight();
```